docs/maui/views/AvatarView.md

```diff
@@ -0,0 +1,360 @@
+---
+title: AvatarView - .NET MAUI Community Toolkit
+author: GeorgeLeithead
+description: "AvatarView is a control for displaying an image or user's initials avatar."
+ms.date: 07/11/2022
+---
+
+# AvatarView
+
+The CommunityToolKit MAUI `AvatarView` displays text and image-based avatars.  Avatars can be text, image, colored, shaped and supports shadow and gestures.
+
+`AvatarView` defines the following properties:
+
+- `BackgroundColor`, of type `Color`, describes the background color of the AvatarView. The default value of this property is White.
+- `BorderColor`, of type `Color`, describes the border color of the AvatarView.
+- `BorderWidth`, of type `double`, defines the width of the AvatarView's border.  The default value for this property is 1.0.
+- `CharacterSpacing`, of type `double`, defines the spacing between characters of the AvatarViews's text.
+- `CornerRadius`, of type `CornerRadius`, which defines the corner radius of the `AvatarView`. This property can be set to a single `double` uniform corner radius value, or a `CornerRadius` structure defined by four `double` values that are applied to the top left, top right, bottom left, and bottom right of the `AvatarView`.  The default value of this property is a device-independent unit of 24.
+- `FontAttributes`, of type `FontAttributes`, determines text style.
+- `FontAutoScalingEnabled`, of type `bool`, defines whether the AvatarView text will reflect scaling preferences set in the operating system. The default value of this property is `true`.
+- `FontFamily`, of type `string`, defines the font family.
+- `FontSize`, of type `double`, defines the font size.  The default value for this property is a device-independent unit of 14.
+- `ImageSource`, of type `ImageSource`, specifies an image to display as the content of the AvatarView.
+- `Padding`, of type `Thickness`, represents the distance between the avatar border and its text or image element.
+- `Text`, of type `string`, defines the text displayed as the content of the AvatarView.  This default value for this property is '?'.
+- `TextColor`, of type `Color`, describes the color of the AvatarView's text.
+- `TextTransform`, of type `TextTransform`, defines the casing of the AvatarView's text.
+
+These properties are backed by `BindableProperty` objects, which means that they can be targets of data bindings and styled.
+
+For information about specifying fonts on an `AvatarView`, see [Fonts](/dotnet/maui/user-interface/fonts.md).
+
+For information about specifying shadows on an `AvatarView`, see [Shadows](/dotnet/maui/user-interface/shadow.md)
```

Comment:
Could we move this to after the section on `Create an AvatarView` so the reader can get a brief overview of how to create the control and then dive in to the ways to customise it?

Reply:
@bijington More than happy with doing that.  Is there an existing example that I could use as a guide?

Reply:
This page might give a good guide where we include a table of properties near the end:

https://docs.microsoft.com/dotnet/communitytoolkit/maui/converters/compare-converter

Reply:
@bijington  I have added a properties section and removed properties that are common control properties.